src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -59,7 +59,7 @@
         "symfony/scheduler": "^6.4.4|^7.0.4",
         "symfony/security-bundle": "^6.4|^7.0",
         "symfony/semaphore": "^6.4|^7.0",
-        "symfony/serializer": "^6.4|^7.0",
+        "symfony/serializer": "^7.1",
```

Comment:
Is that bump really needed? AFAIK, we try to avoid as much as possible to bump dependencies (see https://github.com/symfony/symfony/pull/53160#discussion_r1451510668)

(same for other `composer.json` files)

Reply:
FrameworkBundle often bumps the min version instead of trying to support multiple versions of the service definitions, for simplicity.
This is different for other package, which can be reused outside the fullstack framework and for which supporting compatibility with 2 different major versions of Symfony reduces the risk of dependency hell in those projects (the lower level the package is, the more useful this is).

Reply:
Thank you. 

I've bumped version on FrameworkBundle and HttpKernel. I have reverted the other packages and make sure they support both 6.4 and 7.2.